tests/components/common/flashMessage/messages/EmailConfirmationFailedMessage.spec.tsx

```diff
@@ -14,23 +14,23 @@ describe("EmailConfirmationFailedMessage", () => {
     (useResendConfirmationEmailMutation as jest.Mock).mockImplementation(() => [resendConfirmationEmailMock, { loading: false }]);
   });
 
-  it("renders failed email confirmation message", () => {
+  it("renders failed e-mail confirmation message", () => {
     render(<EmailConfirmationFailedMessage />);
 
-    expect(screen.getByText(/Your email could not be confirmed/)).toBeInTheDocument();
+    expect(screen.getByText(/Your e-mail could not be confirmed/)).toBeInTheDocument();
   });
 
-  it("renders resend confirmation email button", () => {
+  it("renders resend confirmation e-mail button", () => {
     render(<EmailConfirmationFailedMessage />);
 
-    expect(screen.getByRole("button", { name: /Resend Confirmation Email/ })).toBeInTheDocument();
+    expect(screen.getByRole("button", { name: /Resend confirmation e-mail/ })).toBeInTheDocument();
```

Comment:
These email renaming changes should be in their own PR. Having them in a PR (and squashed commit) called "New profile page" doesn't make any sense